config/routes.rb

```diff
@@ -24,7 +24,7 @@
 
   get 'welcome/about'
   get 'welcome/lists'
-  #root 'todo_lists#index'
   root 'welcome#index'
+  get 'houses/index'
```

Comment:
Remove this route.

You already have:  https://github.com/nagano564/house/blob/265482488195cc214ae5d228b7b4d28ac7cf14ae/config/routes.rb#L11-L13

That creates all these routes:

```
➜  house git:(testing) ✗ rake routes | grep house
/Users/harrylevine/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bootstrap-4.0.0.beta/lib/bootstrap/version.rb:2: warning: already initialized constant Bootstrap::VERSION
/Users/harrylevine/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bootstrap-sass-3.3.7/lib/bootstrap-sass/version.rb:2: warning: previous definition of VERSION was here
/Users/harrylevine/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bootstrap-4.0.0.beta/lib/bootstrap/version.rb:3: warning: already initialized constant Bootstrap::BOOTSTRAP_SHA
/Users/harrylevine/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/bootstrap-sass-3.3.7/lib/bootstrap-sass/version.rb:3: warning: previous definition of BOOTSTRAP_SHA was here
                          user_houses GET      /user/:user_id/houses(.:format)                             houses#index
                                      POST     /user/:user_id/houses(.:format)                             houses#create
                       new_user_house GET      /user/:user_id/houses/new(.:format)                         houses#new
                      edit_user_house GET      /user/:user_id/houses/:id/edit(.:format)                    houses#edit
                           user_house GET      /user/:user_id/houses/:id(.:format)                         houses#show
                                      PATCH    /user/:user_id/houses/:id(.:format)                         houses#update
                                      PUT      /user/:user_id/houses/:id(.:format)                         houses#update
                                      DELETE   /user/:user_id/houses/:id(.:format)                         houses#destroy
```

Reply:
And the one you want is the one that goes to the `index` method, which is

```
user_houses GET      /user/:user_id/houses(.:format)                             houses#index
```